server/models/contact.coffee

```diff
@@ -28,6 +28,14 @@ module.exports = class Contact extends cozydb.CozyModel
         # vCard Name = splitted
         # (Familly;Given;Middle;Prefix;Suffix)
         n             : String
+        org           : String
+        title         : String
+        department    : String
+        bday          : String
+        nickname      : String
+        url           : String
+        import        : Boolean
```

Comment:
I should remove it. It's added by the client to tell if the creation comes from an import operation.